addons/point_of_sale/static/src/js/models.js

```diff
@@ -419,6 +419,12 @@ exports.PosModel = Backbone.Model.extend({
                 return new exports.Product({}, product);
             }));
         },
+    },{
+        model: 'account.cash.rounding',
+        fields: ['name', 'rounding'],
```

Comment:
you need to take the one that is set on the pos config. So you need to add a domain to only take the one set on the config